go/vt/vtctl/workflow/materializer.go

```diff
@@ -290,6 +290,15 @@ func (mz *materializer) deploySchema() error {
 		}
 	}
 
+	// Check if any table being moved is already non-empty in the target keyspace.
+	// Skip this check for multi-tenant migrations.
+	if !mz.IsMultiTenantMigration() {
+		err := validateEmptyTables(mz.ctx, mz.ts, mz.tmc, mz.targetShards, mz.ms.TableSettings)
```

Comment:
We're passing in a lot of mz members, which tells me it's probably better to have mz as the method receiver, no?
```
mz.validateEmptyTables()
```

Reply:
I thought about this too, but went with the other approach anyway. Refactored the function with the receiver, it looks much cleaner now!
Also, if the method receiver is `materializer`, it makes sense to have `validateEmptyTables` in `materializer.go`. So, moved it from `utils.go`.